test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Member.Tests.ps1

```diff
@@ -40,6 +40,13 @@ Describe "Get-Member" -Tags "CI" {
 
 	Get-Member -InputObject $o | Should -Not -BeNullOrEmpty
     }
+
+    It "Should be able to be called on IntPtr" {
+        $results = [System.IntPtr] | Get-Member -Type Property -Static | Sort-Object -Property Name
+        $results.Count | Should -BeExactly 2
+        $results[0].Name | Should -BeExactly 'Size'
+        $results[1].Name | Should -BeExactly 'Zero'
```

Comment:
It does not look very reliable. Can Iwe just compare results and expected arrays by Compare-Object?

Reply:
The results are sorted, so it will always be the order `Size, Zero`, unless a new static property is added. But in that case, we will have to update the test anyway.